man/eza.1.md

```diff
@@ -38,6 +38,16 @@ EXAMPLES
 : Displays a tree of files, three levels deep, as well as each file’s metadata.
 
 
+META OPTIONS
+===============
+
+`-?`, `--help`
```

Comment:
We wanna move away from `-?` and to `-h` instead by `v1.0`, maybe we should avoid advertising it.
```suggestion
`--help`
```